### ISSUE TITLE
Rest API: provide example for dataset with with row-id and ignored features.

### ISSUE THREAD

**Issue by mb706** [openml/docs#22]
Not sure where to put this issue, since it is documentation-related on the one hand but not part of the docs.openml.org page on the other.

The json response to `/task/{id}` may sometimes contain the [`ignore_attribute` and `row_id_attribute`](https://github.com/openml/openml.org/blob/ce17242b5bb7dbe1ee9580e841567393a467af85/server/data/views.py#L76-L77) elements, but they are missing for most tasks. This is currently not documented anywhere on the only kind of reference documentation about the REST API that I could find, the "REST" tab of [this page](https://www.openml.org/apis). It would be ideal to have this in the "schema" section, and to also provide an example ID for which they are present in the "Actions" Example Value display.

One dataset that I found that had a `row_id_attribute` was [dataset 164](https://www.openml.org/d/164). 
For `ignore_attribute`, one example is [dataset 185](https://www.openml.org/d/185), and one example with multiple `ignore_attribute` is [dataset 940](https://www.openml.org/d/940).

**Comment by mb706** [comment]
(It would be nice to have a set of IDs for datasets and probably also other entities that cover all of these special issues, so that someone trying to develop against the API could test against these and then be reasonably sure that the implementation is robust.)

**Comment by joaquinvanschoren** [comment]
Thanks! I added a section on special attributes in the new BootCamp docs: https://docs.openml.org/#special-attributes I also included your examples there.

In the spec it is originally part of the `data_features` schema. We also return it (for convenience) in the dataset and task descriptions. You're right that this should also be documented there. 
@PGijsbers Can we include this in the next version of the API schema?
